pandasai/__init__.py

```diff
@@ -86,21 +87,27 @@ def run(
         prompt: str,
         is_conversational_answer: bool = None,
         show_code: bool = False,
+        anonymize_df: bool = False,
```

Comment:
We can also use it as a default value IMO, so I'd set to True by default! What do you think?